core/pom.xml

```diff
@@ -115,12 +115,30 @@
             <scope>provided</scope>
         </dependency>
 
+        <!-- Constructing objects without using constructors -->
+        <dependency>
+            <groupId>org.objenesis</groupId>
+            <artifactId>objenesis</artifactId>
+            <version>2.6</version>
+        </dependency>
+
         <!-- Testing -->
         <dependency>
             <groupId>cglib</groupId>
             <artifactId>cglib-nodep</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <scope>test</scope>
+            <version>4.12</version>
+        </dependency>
```

Comment:
junit isnt required. It comes from the parent pom, as per:
- https://github.com/DozerMapper/dozer/blob/master/plugins-parent/pom.xml#L83-L87